src/request.schema.json

```diff
@@ -18,6 +18,59 @@
   "description": "The HTTP Request",
   "additionalProperties": false,
   "properties": {
+    "url": {
+      "type": "string",
+      "description": "The path and request parameters of the request URL",
+      "examples": [
+        "/index.html?foo=bar"
+      ]
+    },
+    "resourcePath": {
+      "type": "string",
+      "description": "The resource path (without extension) that has been requested",
+      "examples": [
+        "/index"
+      ]
+    },
+    "path": {
+      "type": "string",
+      "description": "The path and request parameters of the request URL",
```

Comment:
This is inconsistent with Express: `path` should not include the query string: https://expressjs.com/en/4x/api.html#req.path